tasks/add_new_openwrap_partner.py

```diff
@@ -475,7 +489,7 @@ def process_price_bucket(self, start_index, end_index, granu):
 
 def setup_partner(user_email, advertiser_name, advertiser_type, order_name, placements,
      sizes, lineitem_type, lineitem_prefix, bidder_code, prices, setup_type, creative_template, num_creatives, use_1x1,
-     currency_code, custom_targeting, same_adv_exception, device_categories, device_capabilities, roadblock_type, slot , adpod_creative_durations, creative_user_def_var):
+     currency_code, custom_targeting, same_adv_exception, device_categories, device_capabilities, roadblock_type, slot , adpod_creative_durations, creative_user_def_var, deal_config, deal_lineitem_enabled):
```

Comment:
Default values can be set to avoid changes at multiple places: 
`deal_lineitem_enabled=None, deal_lineitem_enabled=False`

Ignore if you need to redo lot of changes.

Reply:
Default False are already set at time of variable initialisation.